src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -1,27 +1,33 @@
 {
   "iris.shaders.reloaded": "Shader neu geladen!",
   "iris.shaders.toggled": "Shader umgeschaltet auf: %s",
-  "iris.shaders.toggled.failure": "Shader umschalten fehlgeschlagen! Grund: %s",
   "iris.keybind.reload": "Shader neu laden",
-  "iris.shaders.reloaded.failure": "Shader neu laden fehlgeschlagen! Grund: %s",
-  "iris.keybind.shaderPackSelection": "Shaderpack Auswahlbildschirm",
+  "iris.keybind.shaderPackSelection": "Shaderpaket-Auswahlbildschirm",
   "iris.keybind.toggleShaders": "Shader umschalten",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "Neuladen der Shader fehlgeschlagen! Grund: %s",
+  "iris.shaders.toggled.failure": "Umschalten der Shader fehlgeschlagen! Grund: %s",
+  "iris.sodium.failure.title": "Iris konnte nicht geladen werden!",
+  "iris.sodium.failure.reason": "Sodium befindet sich in deinen Mods-Ordner, wird aber von Iris mitgeliefert. Bitte schließe das Spiel und entferne Sodium aus deinen Mods-Ordner.",
```

Comment:
quick note, it should be "Sodium befindet sich in deinem Mods-Ordner"

Reply:
Thanks! I always mix these two up.